website/content/en/docs/faqs/_index.md

```diff
@@ -16,7 +16,9 @@ You should not have separate logic. Instead design your reconciler to be idempot
 
 Use a [finalizer].
 
-## I keep seeing the following intermittent warning in my Operator's logs: `The resourceVersion for the provided watch is too old.` What's wrong?
+## I keep seeing the following intermittent warning in my Operator's logs: 
+`The resourceVersion for the provided watch is too old.` 
+## What's wrong?
```

Comment:
Is it possible to use the old header, maybe something more succinct like
```suggestion
## I see the warning in my Operator's logs "the resourceVersion for the provided watch is too old". What's wrong?
```

Reply:
![Screen Shot 2020-11-10 at 2 38 42 PM](https://user-images.githubusercontent.com/26195902/98725020-95af6880-2362-11eb-96fe-fa721d0316d3.png)

Updated formatting style to the previous header.